### DISCUSSION TITLE
U3d support

### DISCUSSION THREAD

**Discussion by emanuelhristea** [assimp/assimp#3943] [Q&A] [answered]
Hello everyone, quick question, can this library export files in universal 3d format?

Thanks,
Emanuel

**Comment by kimkulling** [comment, marked as answer]
Not at this moment. You can try to open a feature request for this.

**Reply by emanuelhristea** [reply]
Thanks, I will

**Comment by tellypresence** [comment]
Requested in 
- #3710
